components/app/app.component.tsx

```diff
@@ -1,23 +1,38 @@
 import React from 'react';
-import { HomeComponent } from '@components';
-import Image from 'next/image';
-import { useTranslation } from 'next-i18next';
-import IMG_CONFIG from './image-config';
-const AppComponent: React.FC = () => {
-  const { t } = useTranslation('common');
+
+type AppProps = {
+  about_para: string;
```

Comment:
Why are you taking them as props?

I don't expect app component to change on above props.